tinker/templates/faculty-bios/home.html

```diff
@@ -232,6 +258,64 @@ <h4 class="modal-title" id="myModalLabel">Are you sure you want to delete?</h4>
                 }
             });
         });
+
+
+        $(".courseleaf-user-press").click(function () {
+            $('#spinner').show();
+            var current_element = $(this);
+            var id = current_element.data('faculty-bio-id');
+            var inputs = {
+                "id": id,
+                "added": current_element.attr('data-method')
+            };
+            var url = '/faculty-bios/toggle-courseleaf-user';
+
+            current_element.html('. . . Updating . . .');
+
+            if (current_element.attr('data-method') == 'Yes') {
+                $.notify({
+                    message: 'Adding to Catalog'
+                }, {
+                    type: 'success',
+                    placement: {
+                        align: 'center'
+                    },
+
+                });
+            }
+
+            if (current_element.attr('data-method') == 'No') {
+                $.notify({
+                    message: 'Removing from Catalog'
+                }, {
+                    type: 'success',
+                    placement: {
+                        align: 'center'
+                    },
+
+                });
+            }
+            $.ajax({
+                type: "POST",
+                url: url,
+                data: JSON.stringify(inputs),
+                contentType: 'application/json;charset=UTF-8',
+                success: function (result) {
+                    current_element.html('Success');
+                    if( current_element.attr('data-method') == 'Yes') {
+                        $('#added-text-' + id).text('Added to Catalog');
+                    } else {
+                        $('#added-text-' + id).text('Removed from Catalog');
+                    }
+                    current_element.removeClass('btn-alert');
+                    current_element.css('background-color', '#5F7615');
+                    current_element.css('border-color', '#5F7615');
+                    current_element.css('cursor', 'default');
+                    current_element.prop('onclick',null).off('click');
```

Comment:
I don't love lines 310-314. Do we do that for deactivate?

Reply:
Yes